solutions/silver/usaco-104.mdx

```diff
@@ -2,9 +2,22 @@
 id: usaco-104
 source: USACO Bronze 2012 January
 title: Haybale Stacking
-author: Óscar Garries, Brad Ma
+author: Óscar Garries, Brad Ma, KJ Karaisz
 ---
 
+<Spoiler title="Hint 1">
+Prefix sums consists of an $\mathcal{O}(N)$ preprocess, followed by $\mathcal{O}(1)$ queries.
+</Spoiler>
+
+<Spoiler title="Hint 2">
+Now, what's the *opposite* of that?
+</Spoiler>
```

Comment:
feel like you could combine these two into one

Reply:
^^ -- also kind of a vague hint imo, it would be awesome if you could explain the intuition behind trying this method :)